qa/tasks/mon_thrash.py

```diff
@@ -22,7 +25,7 @@ def _get_mons(ctx):
     mons = [f[len('mon.'):] for f in teuthology.get_mon_names(ctx)]
     return mons
 
-class MonitorThrasher(Thrasher):
+class MonitorThrasher(Greenlet, Thrasher, object):
```

Comment:
I'm not sure you need to inherit from object if since both Greenlet and Thrasher likely do.

Reply:
Also, what's the benefit of making it inherit from Greenlet rather than simply embedding one?

Reply:
Dropped `object`, somehow creep in from the sample code. Regarding `MonitorThrasher`, it was an effort to make the implementation similar to the other thrasher implementations.